Jenkinsfile

```
@@ -236,6 +251,74 @@ pipeline {
                         expression {
                             !skipCompileTests
                         }
+                    }falco
```

```suggestion
                    }
```
I think this is just a typo? It's preventing jenkins from running